packages/extension-browser/src/manifest.json

```diff
@@ -9,8 +9,12 @@
   "icons": {
     "180": "icon.png"
   },
+  "browser_action": {
+      "default_title": "webhint",
+      "default_popup": "browser-action/browser-action.html"
+  },
   "default_locale": "en",
-  "description": "Run webhint in your browser.",
+  "description": "Webhint is a tool for developers that can help you find issues in different areas. Quickly test your website to see if there are any issues with accessibility, browser compatibility, security, performance and more.",
```

Comment:
I changed this so it doesn't look like:

![image](https://user-images.githubusercontent.com/606594/65806019-a2f89c00-e13c-11e9-964e-d414077d4e80.png)

![image](https://user-images.githubusercontent.com/606594/65806033-aee45e00-e13c-11e9-9559-6d046ab6f572.png)